src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json

```diff
@@ -9,9 +32,13 @@
                 "headline": "Current week",
                 "content": "The exercises of the current week will be displayed on top of the exercise list."
             },
+            "currentExercise": {
+                "headline": "Current Exercise",
+                "content": "In this tutorial we will have a look at the 'Hello World' exercise."
+            },
             "exerciseType": {
                 "headline": "Exercise type",
-                "content": "The exercise type can be distinguished by the exercise icon."
+                "content": "The exercise type can be distinguished by the exercise icon. This example here is a programming exercise."
             },
             "exerciseTags": {
                 "headline": "Exercise tags",
```

Comment:
In the lectures it says: `You can find and download the all lecture slides from this tab.` should be either `all lecture slides` or `all of the lecture slides`, right?

Reply:
You forgor a dot in this one: `The listed tags will give further information on the exercise classification, e.g. by difficulty`